cv_bridge/src/CMakeLists.txt

```diff
@@ -38,7 +39,7 @@ if(NOT ANDROID)
   endif()
 
   set_target_properties(${PROJECT_NAME}_boost PROPERTIES
-                        LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/boost/
+                        LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/python/ament_cmake_python/${PROJECT_NAME}/${PROJECT_NAME}/boost
```

Comment:
shouldn't you use `${PYTHON_INSTALL_DIR}/${PROJECT_NAME}/boost/` instead?

Reply:
I change to refer to the python library under install at https://github.com/ros-perception/vision_opencv/pull/444/commits/fe5d6eddd1d9bcb3a9f40fc9caa7395b7c7c7c9d.